app/grandchallenge/algorithms/templates/algorithms/job_detail.html

```diff
@@ -12,7 +12,7 @@
 {% load civ %}
 
 {% block title %}
-    {{ object.pk }} - Results - {{ block.super }}
+    Result {{ object.pk }} - {{ object.algorithm_image.algorithm.title }} - {{ block.super }}
```

Comment:
This is introducing inconsistencies between the title and breadcrumbs again. I thought we wanted them to be the same? I see that with a maximum depth of 3, you wouldn't be able to include the algorithm title in the title here, but then maybe do: `{{ object }} - {{ object.algorithm_image.algorithm.title }} - {{ block.super }}` and change the last breadcrumb to `{{ object }}` too? That will print (if I'm not mistaken, do double check) "Job 1234".

Reply:
Here, {{ object }} will show Job instead of results which was pointed before as inconsistent.
See comment above. I will stick to fixing the casing and incorrect things to remain consistent with the breadcrumbs.

Reply:
Ok, but on the Job Progress template you do use the term "Job" instead of result, so that needs to be changed then too, I think.

Reply:
Indeed, I will delete it, "Progress" will do fine I think.